### DISCUSSION TITLE
Need more information/documentation around what lora specific hyperparameters mean

### DISCUSSION THREAD

**Discussion by nayan-dhabarde** [huggingface/peft#1174] [Q&A]
I have used dreambooth for training in the past and that worked really well for me.
It looks like same hyperparameter values wont work in case of Lora.
Additionally, I don't know what these parameter do
--lora_r 16 \
        --lora_alpha 27 \
        --lora_text_encoder_r 16 \
        --lora_text_encoder_alpha 17

Can someone help my understand how increasing this or decreasing this will affect the results
Also, when training for faces, what is best learning rate, lr schedule value and number of steps
I am using 2000 steps with 2000 reg images and 10 input imagesz

**Comment by younesbelkada** [comment]
Indeed choosing the right hyper-parameters when using LoRA is a very important topic. Another hyper-parameter you can use is the weights initialization strategy introduced by: https://github.com/huggingface/peft/pull/1189 We found out that when using `"gaussian"` you can use smaller LR (similarly as in `diffusers` )
